### DISCUSSION TITLE
Amazon.Lambda.Annotations with Terraform

### DISCUSSION THREAD

**Discussion by matiaspecile** [aws/aws-lambda-dotnet#1693] [Q&A] [answered]
Hi,

The Amazon.Lambda.Annotations documentation states that only CloudFormation deployments are currently supported:

_Lambda Annotations can be added to existing projects. Lambda Annotations does require that deployment of existing projects is done using a CloudFormation template. In the future Lambda Annotations may support other deployment technologies._

Is there any progress towards providing support for other deployment methods (Terraform specifically for my use case)?  Is there any way to make this work with Terraform today, even if the approach is of an experimental nature?

**Comment by ashishdhingra** [comment]
@matiaspecile Good afternoon. As of now, other deployment methods are not supported. Let me review this with the team.

Thanks,
Ashish

**Reply by ashishdhingra** [reply, marked as answer]
Discussed with the team. Converting this discussion to feature request https://github.com/aws/aws-lambda-dotnet/issues/1704 (although we don't have an ETA as of now).
